Hunted Mobile/Hunted Mobile/Hunted Mobile/Service/HttpClientResponse.cs

```diff
@@ -52,7 +52,7 @@ public class HttpClientResponse {
 
         protected async Task ReadingRequestContent() {
             try {
-                _responseContent = await ResponseMessage.Content.ReadAsStringAsync();
+                ResponseContent = await ResponseMessage.Content.ReadAsStringAsync();
             }
             catch(Exception e) {
                 MainErrorMessage = "Er is iets misgegaan bij het uitlezen van de inhoud van de response";
```

Comment:
Zijn deze error messages bestemd voor de gebruiker? Zo niet dan horen ze eigenlijk Engels te zijn.

Reply:
Deze class is van @Ghoster524 , volgens mij.

Reply:
Dan wellicht de error-meldingen eruit halen.